ledger-service/http-json-testing/src/main/scala/com/daml/http/WebsocketTestFixture.scala

```diff
@@ -176,25 +176,24 @@ private[http] object WebsocketTestFixture extends StrictLogging with Assertions
       jwt: Jwt,
       serviceUri: Uri,
       query: String,
-      offset: Option[domain.Offset] = None,
-      keepOpen: Boolean = false)(implicit asys: ActorSystem): Source[Message, NotUsed] =
-    singleClientWSStream(jwt, "query", serviceUri, query, offset, keepOpen)
+      offset: Option[domain.Offset] = None)(implicit asys: ActorSystem): Source[Message, NotUsed] =
```

Comment:
akka-http cancels the output section of the stream if the input is closed which is why we need to keep it open. So far, we got away with not doing that in a lot of places but now you get an exception if you hit this, so we just always keep it open and terminate the output via killswitches or `take`.